wies/projects/templates/placement_list.html

```diff
@@ -1,119 +1,620 @@
+                      role="tab"
+                      aria-selected="true"
+                    >
+                      Inzet
+                    </a>
+                  </li>
+                  <li role="presentation" class="rvo-tabs__item">
+                    <a
+                      href="#panel-beschikbaarheid"
+                      class="rvo-link rvo-tabs__item-link rvo-link--no-underline rvo-link--normal"
+                      role="tab"
+                      aria-selected="false"
+                    >
+                      Beschikbaarheid
+                    </a>
+                  </li>
+                </ul>
+                
+                <div class="rvo-tabs__panel rvo-tabs__panel--active" role="tabpanel" id="panel-inzet" aria-labelledby="tab-inzet">
+                  <!-- Results Table -->
+                  <div id="placement_table" class="rvo-table-container">
+                    {% include 'parts/placement_table.html' %}
+                  </div>
+                </div>
+                
+                <div class="rvo-tabs__panel" role="tabpanel" id="panel-beschikbaarheid" aria-labelledby="tab-beschikbaarheid" style="display: none;">
+                  <!-- Beschikbaarheid content -->
+                  <div id="timeline-container">
+                    {% include 'parts/placement_timeline.html' %}
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
   </div>
-</div>
+</body>
+
+<script>
+document.addEventListener('DOMContentLoaded', function() {
+  const tabs = document.querySelectorAll('.rvo-tabs__item-link');
+  const panels = document.querySelectorAll('.rvo-tabs__panel');
+  
+  tabs.forEach(tab => {
+    tab.addEventListener('click', function(e) {
+      e.preventDefault();
+      
+      // Remove active class from all tabs and panels
+      tabs.forEach(t => {
+        t.classList.remove('rvo-tabs__item-link--active', 'rvo-link--active');
+        t.classList.add('rvo-link--normal');
+        t.setAttribute('aria-selected', 'false');
+      });
+      
+      panels.forEach(p => {
+        p.classList.remove('rvo-tabs__panel--active');
+        p.style.display = 'none';
+      });
+      
+      // Add active class to clicked tab
+      this.classList.add('rvo-tabs__item-link--active', 'rvo-link--active');
+      this.classList.remove('rvo-link--normal');
+      this.setAttribute('aria-selected', 'true');
+      
+      // Show corresponding panel
+      const targetId = this.getAttribute('href').substring(1);
+      const targetPanel = document.getElementById(targetId);
+      if (targetPanel) {
+        targetPanel.classList.add('rvo-tabs__panel--active');
+        targetPanel.style.display = 'block';
+      }
+    });
+  });
+});
+</script>
+
 
 <style>
-.view-switcher {
+
+/* Compact filter styling */
+.rvo-filter .utrecht-form-field {
```

Comment:
in other cases we have split the css from the html, curious if there is a specific reason why you have chosen to include it here?